File: app/src/main/java/com/alyndroid/pokemon/db/PokemonDB.java
package com.alyndroid.pokemon.db;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.alyndroid.pokemon.model.Pokemon;

@Database(entities = Pokemon.class, version = 1, exportSchema = false)
public abstract class PokemonDB extends RoomDatabase {
    public abstract PokemonDao pokemonDao();
}

File: app/src/main/java/com/whiteside/bokemoncourse/db/PokemonDB.kt
package com.whiteside.bokemoncourse.db

import androidx.room.Database
import androidx.room.RoomDatabase
import com.whiteside.bokemoncourse.model.Pokemon

@Database(entities = [Pokemon::class], version = 1, exportSchema = false )
abstract class PokemonDB : RoomDatabase() {

    abstract fun getDao() : PokemonDao;
}